src/maintainer/pinning_deps.rst

```diff
@@ -3,6 +3,20 @@
 Pinned dependencies
 *******************
 
+**Pinning a dependency** is a practice of explicitly pointing to the version of a library which our package is linked with. 
+A pinned dependency , for example, can look like this : ``numpy==1.1.3`` . On the other hand, a non-pinned dependency would be  just: ``numpy``.
+
+**Why to pin your dependencies?**
+
+Explicitly declaring the versions of the dependencies could be advantageous to the quality of the software and to the developers and the open source community that makes up the software ecosystem.
+ 
+* Pinning our dependencies may help in avoiding a situation where our software does not builds or runs due to the release of newer versions of the dependencies which are incompatible with our software, consisting some breaking changes.
+* Not updating the pinned dependencies each time we upgrade our software and deploy it with newer versions of those dependencies, can result in an older version to hang around longer than it should, which can pose difficulties if these older version have some security issues. These older versions might also be incompatible with some new dependency that is introduced.
```

Comment:
This part is kind of confusing

Reply:
The whole part, or just the second point? "Not updating the pinned dependencies...." 
@viniciusdc

Reply:
the second one